build/templates/setup.py.mako

```diff
@@ -2,7 +2,34 @@
 # -*- coding: utf-8 -*-
 # This file was generated
 <%
-config        = template_parameters['metadata'].config
+config         = template_parameters['metadata'].config
+module_version = config['module_version']
+
+from packaging.version import Version
```

Comment:
you should factor out a function that determines the dev_status, so we can unit test.

IMO:

1 - Planning: Starting at 0.0
2 - Pre-Alpha: Starting at 0.3
3 - Alpha: Starting at 0.5
4 - Beta: Starting at 0.9
5 - Production/Stable Starting at 1.0
6 - Mature N/A
7 - Inactive N/A

Reply:
You think we need this level of granularity?

What about one the driver has gone to 1.0? Do we still need `Planning` and `Pre-Alpha`? See PR description for current algorithm for drivers > 1.0.

Reply:
Moved into separate function and tested. Did not add all the granularity. Waiting on comment.

Reply:
I think how we have it is fine.